src/test/java/spoon/test/imports/ImportTest.java

```diff
@@ -71,7 +71,9 @@ public void testImportOfAnInnerClassInASuperClassAvailableInLibrary() throws Exc
 		String expected = "visibility.YamlRepresenter.RepresentConfigurationSection";
 		assertEquals(expected, innerClass.getReference().toString());
 
-		expected = "org.yaml.snakeyaml.representer.Representer.RepresentMap";
+		// was org.yaml.snakeyaml.representer.SafeRepresenter.RepresentMap
+		// but RepresentMap is defined in SafeRepresenter
```

Comment:
Yes but `SafeRepresenter` isn't public. We cannot print `org.yaml.snakeyaml.representer.SafeRepresenter.RepresentMap` because the Java source code wont be valid.